selfdrive/ui/qt/maps/map_settings.cc

```diff
@@ -244,11 +249,11 @@ DestinationWidget::DestinationWidget(QWidget *parent) : QPushButton(parent) {
   inner_frame->setSpacing(0);
   {
     title = new ElidedLabel(this);
-    title->setAttribute(Qt::WA_TransparentForMouseEvents);
+//    title->setAttribute(Qt::WA_TransparentForMouseEvents);
```

Comment:
@incognitojam do you know why we need these?

Reply:
if you tap on the text the click event doesn't get passed to the DestinationWidget without it

Reply:
https://github.com/commaai/openpilot/pull/28869/files